### DISCUSSION TITLE
Typo: "Total Number of Drives Installed" should be: "Total Number of Drivers Installed"

### DISCUSSION THREAD

**Discussion by SkybuckFlying** [turkaysoft/glow#44] [Ideas]
Typo, see title.

It's in GUI and Report as well.

**Comment by turkaysoft** [comment]
> Typo, see title.
> 
> It's in GUI and Report as well.

Thank you for your suggestion and for letting me know.

**Comment by turkaysoft** [comment]
> Typo, see title.
> 
> It's in GUI and Report as well.

The necessary edit has been made with version v24.14, thanks.
